eldritch/test_mythos.py

```diff
@@ -65,6 +66,7 @@ def testOpenGate(self):
     self.assertEqual(self.state.characters[0].place.name, "Diner")
     self.assertEqual(self.state.characters[1].place.name, "Pluto1")
     self.assertEqual(self.state.characters[2].place.name, "Pluto1")
+    self.assertEqual(self.state.ancient_one.doom, 1)
```

Comment:
Can you add a few more tests for the doom token count? One for a sealed gate, and one more in the monster surge test?